eagle-security/eagle-security-hdfs-auditlog/src/test/java/org/apache/eagle/security/auditlog/TestSiddhiPattern.java

```diff
@@ -0,0 +1,103 @@
+
```

Comment:
The license information is missing. 
btw: We might also check in the eclipse format/template into code base.